packages/api/src/controllers/timeNotificationController.js

```diff
@@ -60,6 +60,40 @@ const timeNotificationController = {
       return res.status(400).send({ error });
     }
   },
+
+  /**
+   * Notifies a user of tasks due today
+   * @param {Request} req request
+   * @param {Response} res response
+   * @async
+   */
+  async postDailyDueTodayTasks(req, res) {
+    const { farm_id } = req.params;
+    try {
+      const activeUsers = await UserFarmModel.getActiveUsersFromFarmId(farm_id);
+
+      if (activeUsers.length) {
+        const tasksDueTodayNotificationUsers = [];
+        for (const { user_id } of activeUsers) {
+          const { rows: tasksDueToday } = await TaskModel.getTasksDueTodayFromUserId(user_id);
```

Comment:
The function called here needs to check for tasks due today for `user_id` on `farm_id`. Otherwise line 81 may be notifying about user's tasks on some other farm.